python/cudf/cudf/core/series.py

```diff
@@ -3778,7 +3778,11 @@ def describe_numeric(self):
             data = _format_stats_values(data)
 
             return Series(
-                data=data, index=names, nan_as_null=False, name=self.name
+                data=data,
+                index=names,
+                nan_as_null=False,
+                name=self.name,
+                dtype="float",
```

Comment:
What happens if we don't specify `dtype="float"`? Is it not properly inferred anymore or is this just to short circuit some unnecessary code?

Reply:
It fails in case like this 
```
cudf.Series([np.int32(4), np.float64(1.5), np.float32(1.290994), np.int8(0)])

pyarrow.lib.ArrowInvalid: Integer scalar type not recognized

```
but not in 
```
>>> cudf.Series([4, 1.5, 1.290994, 0])
0    4.000000
1    1.500000
2    1.290994
3    0.000000
dtype: float64

```

Reply:
I would expect `[np.int32(4), np.float64(1.5), np.float32(1.290994), np.int8(0)]` to be handled by falling back to a numpy array and end up as a `float64` dtype column. Did we handle this before these changes?

Reply:
numpy would handle if we don't expect it to dtype to 'object'

Reply:
Yea the flow I generally expected is: try using pyarrow, on failure fall back to numpy, on a numpy object type, try constructing a pyarrow array from it which should handle converting to either String or List type as needed, or somewhat nicely throw on mixed dtypes